/**: # wildcard for the node name
  ros__parameters:
    feedthrough: false
    gains:
      roll:
        p: 1.0
        i: 0.0
        d: 0.0
        feed_forward: 0.0
      pitch:
        p: 1.0
        i: 0.0
        d: 0.0
        feed_forward: 0.0
      yaw:
        p: 1.0
        i: 0.0
        d: 0.0
        feed_forward: 0.0
    integral_limits:
      roll: 0.5
      pitch: 0.5
      yaw: 0.5
    zero_integral_threshold: 2.0
